### PR TITLE
Fix incorrect 3.2.1 refs

### PR DESCRIPTION
**Pull request by patrick-stephens**
Manual update in #6 left in some 3.2.1 references.

Copied over 3.2.0 formula to ensure is correct.

### CODE REVIEW

File: Formula/calyptia.rb [hunk 1]
```diff
@@ -2,14 +2,14 @@
 # frozen_string_literal: true
 
 # This file was generated by GoReleaser. DO NOT EDIT.
-class Calyptia < Formula
+class CalyptiaAT320 < Formula
```
Comment (nicolasparada) [inline Formula/calyptia.rb:5]: Why is it called like that "CalyptiaAT320"?

**Review by nicolasparada — APPROVED** [review]

File: Formula/calyptia.rb (hunk 1 above)
Comment (patrick-stephens) [reply Formula/calyptia.rb]: Yeah that was a copy-paste error I resolved

File: Formula/calyptia@3.rb [hunk 2]
```diff
@@ -2,14 +2,14 @@
 # frozen_string_literal: true
 
 # This file was generated by GoReleaser. DO NOT EDIT.
-class Calyptia < Formula
+class CalyptiaAT3 < Formula
```
Comment (nicolasparada) [inline Formula/calyptia@3.rb:5]: Is this one good too "CalyptiaAT3"?

Comment (patrick-stephens) [reply Formula/calyptia@3.rb:5]: yup, it was wrong before - it should match up with the filename and I checked the v1/v2 ones to confirm.

**Pull request merged**